src/convert.rs

```diff
@@ -447,7 +447,13 @@ pub struct GlobalStack {
 }
 
 const GLOBAL_STACK_CAP: usize = 16;
-static mut GLOBAL_STACK: [u32; GLOBAL_STACK_CAP] = [0; GLOBAL_STACK_CAP];
+
+// Increase the alignment to 8 here because this can be used as a
+// BigUint64Array pointer base which requires alignment 8
+#[repr(align(8))]
```

Comment:
Apparently the original change in this PR caused the `u64::works` test to fail due to the pointer being returned by `__wbindgen_global_argument_ptr` being aligned to 4 instead of 8. We use this pointer as the base of a `BigUint64Array`, however, so we needed a bigger alignment on global. AKA a preexisting bug!

Presumably this was uncovered because the function changes below added a static which tweaked alignment of fields in this crate, but just wanted to make sure this wasn't thought as being totally random!